src/lib/components/ui/ArticleCard.svelte

```diff
@@ -2,12 +2,83 @@
 	import type { ArticleMetadata } from '$lib/types/article';
 	import { slide } from 'svelte/transition';
 	import Badge from './badge/badge.svelte';
+	import { colorRegex } from '$lib/types/article';
+	import { writable } from 'svelte/store';
 
+	const containerClasses = writable<string>('');
```

Comment:
writables are svelte 4 way to create global stores. You don't want a store, you want a local state and the way you declare a state in svelte 5 is using`$state`